### PR TITLE
DEMO-92: Render the Form UI as per the designs for 1/2/3 column

### PR DESCRIPTION
**Pull request by sunpietro**
https://jira.ez.no/browse/DEMO-92

## Form with no errors
form no errors
## Form with errors
form with errors

### CODE REVIEW

**Comment by sunpietro** [comment]
requires: ezsystems/ezstudio-demo-bundle#184 and ezsystems/ezstudio-form-builder#148

File: app/Resources/EzSystemsFormBuilderBundle/views/fields/captcha/basic.html.twig [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+    <div class="ezform-field-group" data-error-message="{{ 'field.is.required'|trans }}">
+        <span class="ezform-field-indicator"></span>
+        <span class="ezform-field-wrapper">
+            <input
+                id="{{ field.id }}"
+                type="text"
+                class="ezform-field ezform-captcha form-control"
+                placeholder="{{ field.placeholderText }}"
+                {% if field.isRequired %}required{% endif %}
+                >
```
Comment (kamilmusial) [inline app/Resources/EzSystemsFormBuilderBundle/views/fields/captcha/basic.html.twig]: Q: Is this indentation correct?

Comment (sunpietro) [inline app/Resources/EzSystemsFormBuilderBundle/views/fields/captcha/basic.html.twig]: Yes, it is. The React way.

Comment (andrerom) [inline app/Resources/EzSystemsFormBuilderBundle/views/fields/captcha/basic.html.twig]: but this is Twig :)

Comment (sunpietro) [inline app/Resources/EzSystemsFormBuilderBundle/views/fields/captcha/basic.html.twig]: @andrerom of course. I thought it's more readable this way, when a tag has a lot of attributes inside.

**Pull request merged**